Classes/Hooks/HookService.php

```diff
@@ -0,0 +1,114 @@
+<?php
+declare(strict_types=1);
+
+namespace OliverKlee\Seminars\Hooks;
+
+use UnexpectedValueException;
+
+use OliverKlee\Seminars\Interfaces\Hook;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * This class provides functions for unified hooks.
+ *
+ * It unifies the wide-spread use of this type of hooks in seminars.
+ *
+ * @author Michael Kramer <m.kramer@mxp.de>
+ */
+class HookService
+{
+    /**
+     * Interface name to this hook
```

Comment:
Could you please drop all field comments that don't add any information in addition to what already is communicated by the field name?